Rakefile

```diff
@@ -28,6 +28,13 @@ RSpec::Core::RakeTask.new(:spec) do |t, args|
   t.rspec_opts = args.to_a.join(' ')
 end
 
+namespace :spec do
+  RSpec::Core::RakeTask.new(:stress_tests) do |t, _args|
+    t.pattern = 'spec/**/*_spec.rb'
+    t.rspec_opts = '--tag stress_tests'
+  end
+end
+
```

Comment:
Please move namespace definition to all other namespace definitions

Reply:
Fixed in 7ccc0aeba56ee3a80c72fa056f86c85576afd276